kong/plugins/ai-prompt-decorator/handler.lua

```diff
@@ -0,0 +1,67 @@
+local _M = {}
+
+-- imports
+local kong_meta      = require "kong.meta"
+local access_handler = require("kong.plugins.ai-prompt-decorator.access")
```

Comment:
This pattern of putting plugin code in `access.lua` is leftover from older versions of Kong that expected plugins to have a different file tree structure. Nowadays it is preferred to just put your code in `handler.lua`